app/javascript/components/rooms/RoomsList.jsx

```diff
@@ -1,42 +1,49 @@
-import React from 'react';
-import {
-  Row, Col, Button, Spinner, Form, FormControl,
-} from 'react-bootstrap';
+import React, { useState } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import Spinner from '../shared/stylings/Spinner';
 import RoomCard from './RoomCard';
 import useRooms from '../../hooks/queries/rooms/useRooms';
 import ButtonLink from '../shared/stylings/buttons/ButtonLink';
 
 export default function RoomsList() {
   const { isLoading, data: rooms } = useRooms();
+  const [search, setSearch] = useState('');
   if (isLoading) return <Spinner />;
 
   return (
     <>
       <Row className="mt-4 mb-4">
         <Col>
-          {/* TODO: May need to change search form when implementing search functionality */}
-          <Form className="d-flex">
-            <FormControl
-              type="search"
-              placeholder="Search"
-              className=""
-              style={{ width: '19rem' }}
-              aria-label="Search"
-            />
-            <Button className="ms-2" variant="outline-secondary">Search</Button>
-          </Form>
+          <input
```

Comment:
Might as well use the FormControl component instead of input for consistency. 
Should be useful when we add the search icon inside the input field.. which doesn't seem to be supported by React Bootstrap for some reason.